src/vs/workbench/contrib/search/browser/searchModel.ts

```diff
@@ -1608,6 +1608,7 @@ export class SearchResult extends Disposable {
 		super();
 		this._rangeHighlightDecorations = this.instantiationService.createInstance(RangeHighlightDecorations);
 
+		this.modelService.getModels().forEach(model => this.onModelAdded(model));
```

Comment:
fyi @andreamah @roblourens

Reply:
That seems unnecessary because we will do this when the search runs? But is fine